src/components/Name.tsx

```diff
@@ -30,10 +33,18 @@ export function TypewriterHeading({
     }
   }, [currentIndex, delay, text, wait]);
 
-  return <h1 className="text-9xl font-bold">{currentText}</h1>;
+  return (
+    <h1 className={`${isSmall ? 'text-6xl' : 'text-9xl'} font-bold`}>
```

Comment:
This only works because `isSmall` is a static prop- normally I would not recommend conditional tailwind classes because of how they compile

Reply:
oh rad- thanks for flagging that `cn` helps w/ compilation